app/src/util/api.ts

```diff
@@ -69,7 +69,7 @@ export function apiHandler(handler: NextHandler) {
 
 function errorHandler(err: unknown, _req: NextRequest) {
   // TODO log structured request info like route here
-  console.error(err);
+  logger.error(err);
```

Comment:
The TODO comment can perhaps be removed as the logger is automatically logging more request context?